python_modules/dagster/dagster/_core/blueprints/load_from_yaml.py

```diff
@@ -105,10 +111,30 @@ def load_defs_from_yaml(
     else:
         file_paths = list(resolved_path.rglob("*.yaml")) + list(resolved_path.rglob("*.yml"))
 
-    blueprints = [
-        parse_yaml_file_to_pydantic(per_file_blueprint_type, file_path.read_text(), str(file_path))
-        for file_path in file_paths
-    ]
+    origin = get_origin(per_file_blueprint_type)
+    if safe_is_subclass(origin, Sequence):
+        args = get_args(per_file_blueprint_type)
+        if not args:
+            raise ValueError("Sequence type annotation must have a single type argument")
```

Comment:
We don't normally raise `ValueError`s.  Should either be a DagsterError subclass or a CheckError.

Reply:
Updated to check error